projects/k8s-semconv.md

```diff
@@ -0,0 +1,96 @@
+# K8s Conventions Working Group
```

Comment:
Could this be adapted to follow the current project template in https://github.com/open-telemetry/community/blob/main/project-template.md?